src/components/Header/TopNav/ServicesSpecialitiesMap.js

```diff
@@ -7,34 +9,43 @@ const specialitiesMap = {
   openSource: []
 }
 
-const servicesColors = {
-  default: 'black',
-  defaultText: 'white',
+const logoColors = {
+  default: theme.colors.text,
+  defaultText: theme.colors.white,
   defaultHover: '#8e8e8e',
-  engineering: '#52FFAC',
-  design: '#52FFAC',
-  training: '#52FFAC',
-  delivery: '#52FFAC',
-  dedicatedTeams: '#52FFAC',
-  openSource: '#52FFAC',
-  specialityText: '#090329',
-  specialityHover: 'white'
+  engineering: {
+    'node-js': '#52FFAC',
```

Comment:
The `logoColor` for each speciality page is a value which comes from Contentful. So I reckon it's probably not ideal to be hard-coding them in here & duplicating the "source of truth", as it were.

We also want to be getting to a point where Greta can add new speciality pages without consulting an engineer. So ideally, we wouldn't have the list of specialities/services hard-coded into the codebase either.

That one is a little more work, given that the TopNav now relies on `specialitiesMap`, so can probably be dealt with as part of a separate ticket. But off the top of my head, one solution would be to move the navbar outside of `Layout`, since this component needs to change its contents, depending on which page its on.

Reply:
I didn't know; but now that I do, it makes perfect sense. You are right this should probably be part of another ticket. Shall I create one or do you want to? Once this is done is this PR ready to go?

Reply:
Well there's 2 things here:
- the logo colours are now being hard-coded (part of this PR)
- this file hard-codes the names of the specialities (can be part of another ticket)

Reply:
Oh no. My bad, both are already hard-coded on master & this PR is just moving the location into an object called `engineering`.

Yeah, it can all go in a separate ticket then :+1: